src/geode/geometry/distance.cpp

```diff
@@ -26,12 +26,33 @@
 #include <geode/basic/logger.h>
 
 #include <geode/geometry/basic_objects.h>
+#include <geode/geometry/perpendicular.h>
 #include <geode/geometry/projection.h>
+#include <geode/geometry/signed_mensuration.h>
 #include <geode/geometry/vector.h>
 
 namespace
 {
     constexpr double MAX_DOUBLE = std::numeric_limits< double >::max();
+
+    bool point_is_in_triangle(
```

Comment:
Should we rename this in something with approx in the name? because this function should not be used for exact answer. How about `may_point_is_in_triangle`? @panquez what do you think?

Reply:
am not sure, people do not read comment but they have to read the function name :p

Reply:
in any case the function is not exported